app/access/service_key_access.rb

```diff
@@ -14,5 +14,11 @@ def read?(service_key)
       return true if admin_user?
       service_key.service_instance.space.developers.include?(context.user)
     end
+
+    def index?(_, params=nil)
+      return true if admin_user?
+      return true unless params && params.key?(:related_obj)
```

Comment:
Could you explain more about this line? It looks like we'll continue to the next line in the case of developer/manager/auditor role. There isn't any test coverage for the case where we return true, so we don't understand what kind of user it would apply to. Did you mean to put false here?